packages/@lwc/integration-not-karma/README.md

```diff
@@ -1,3 +1,96 @@
-# @lwc/integration-not-karma
+# LWC [Web Test Runner](https://modern-web.dev/docs/test-runner/overview/) integration tests
 
-It's not karma, it's something else!
+## Quick Start
+
+To run integration tests, run `yarn test:wtr` from the monorepo root, or `yarn test` from the package directory.
+
+To run hydration tests, run `yarn test:hydration` from either the monorepo root or the package directory.
+
+To manually debug tests in your browser, add the `--manual` flag to the test command.
+
+To run individual test files, provide them as CLI arguments. If using relative paths, they must be relative to the _package directory_, e.g. `yarn test:wtr test/act/index.spec.js`.
+
+Environment variables are used as controls to run tests in different modes (e.g native vs synthetic shadow, different API versions). The full list of controls is defined in [`helpers/options.js`](./helpers/options.js).
+
+## Architecture
+
+- `configs`: WTR configuration files. The main entrypoints are `integration.js` and `hydration.js`.
+- `helpers`: Helper functions used by tests and the test runner.
+- `mocks`: Module mocks to replace imports in tests.
+- `test`: The test directory for integration tests.
+- `test-hydration`: The test directory for hydration tests.
+
+### Integration Tests
+
+Integration tests are simply `.spec.js` files that run in the browser. LWC components are transformed by a plugin defined in `serve-integration.js`.
+
+### Hydration Tests
+
+Hydration tests test the SSR packages, and are therefore more complex than the integration tests. While the files are named `index.spec.js`, they are actually _config_ files. The actual test executed is defined in `test-hydration.js`. The hydration tests are transformed by a plugin defined in `serve-hydration.js`.
+
+Each hydration test is expected to define an entrypoint component named `x/main`. Each test config is expected to adhere to this interface:
+
+```ts
+interface HydrationTestConfig<Snapshot extends any> {
```

Comment:
Is there somewhere we can reference the interface in code?

If not, should we indicate that this read me needs to be updatd in the config ever changes?